docker-app/qfieldcloud/core/tests/test_packages.py

```diff
@@ -534,3 +542,45 @@ def test_filename_with_whitespace(self):
                 "project_qfield.qgs",
             ],
         )
+
+    def test_offline_non_geopackage(self):
+        exts = ["qgs", "shp", "cpg", "dbf", "prj", "shx"]
+        files = [
+            (
+                f"delta/project_with_offline_nongpkg.{ext}",
+                f"project_with_offline_nongpkg.{ext}",
+            )
+            for ext in exts
+        ]
+
+        # When user has an account that supports non-geopackages layers, it works
+        self.user1.useraccount.account_type = AccountType.objects.create(
+            code="with_nongpkg",
+            display_name="with_nongpkg",
+            is_nongpkg_supported=True,
+        )
+        self.user1.useraccount.save()
+        self.upload_files(self.token1.key, self.project1, files=files)
+        status = self.wait_for_project_status(self.project1)
+        self.assertEqual(status, Project.Status.OK)
+        last_process_job = Job.objects.filter(
+            project=self.project1, type=Job.Type.PROCESS_PROJECTFILE
+        ).latest("updated_at")
+        self.assertEqual(last_process_job.status, Job.Status.FINISHED)
+
+        # When user has an account that does not supports non-geopackages layers, it fails
+        self.user1.useraccount.account_type = AccountType.objects.create(
+            code="without_nongpkg",
+            display_name="without_nongpkg",
+            is_nongpkg_supported=False,
+        )
+        self.user1.useraccount.save()
+        self.upload_files(self.token1.key, self.project1, files=files)
+        status = self.wait_for_project_status(self.project1)
+        # TODO: REVIEW: the job fails (see below), but the project status is still ok !
+        # Is this expected ? What does project.status actually represent ?
+        # self.assertEqual(status, Project.Status.FAILED)
```

Comment:
@suricactus Can you check that ? Is that expected ?

Reply:
I close it because I don't know if this is still the issue.